tests/integration/test_backup_restore_on_cluster/test.py

```diff
@@ -1054,9 +1054,12 @@ def test_mutation():
     backup_name = new_backup_name()
     node1.query(f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name}")
 
-    assert not has_mutation_in_backup("0000000000", backup_name, "default", "tbl")
+    # mutation #0000000000: "UPDATE x=x+1 WHERE 1" could already finish before starting the backup
```

Comment:
instead of this check you can verify that all updates are applied after restore and mutations finish?

Reply:
Yes, I agree that it would be better. But unfortunately that won't work because mutations are not recreated by `RESTORE`. Though they are written to a backup. There is a plan to make `RESTORE` recreate mutations, and even some work was done to do that, but I haven't finished it yet (https://github.com/ClickHouse/ClickHouse/pull/50230).